packages/client-core/src/admin/components/Party/PartyTable.tsx

```diff
@@ -49,10 +54,17 @@ const PartyTable = (props: PartyPropsTable) => {
       action: (
         <>
           <a href="#h" className={classes.actionStyle}>
-            {' '}
-            <span className={classes.spanWhite}>View</span>{' '}
+            <span className={classes.spanWhite}>View</span>
```

Comment:
Need to implement this functionality

Reply:
I am working on it

Reply:
If you are still working on the PR please convert it to a draft PR